RubberduckTests/Settings/HotkeySettingsTests.cs

```diff
@@ -92,5 +92,77 @@ public void DuplicateNamesAreIgnored()
 
             Assert.IsFalse(settings.Settings.Contains(duplicate));
         }
+
+        [TestMethod]
+        [TestCategory("Settings)")]
+        public void HotkeyModifier_None_IsNotValid()
+        {
+            var setting = new HotkeySetting { Name = "Foo", Key1 = "I" };
+
+            Assert.IsFalse(setting.IsValid);
+        }
+
+        [TestMethod]
+        [TestCategory("Settings)")]
+        public void HotkeyModifier_Shift_IsNotValid()
+        {
+            var setting = new HotkeySetting { Name = "Foo", HasShiftModifier = true, Key1 = "I" };
+
+            Assert.IsFalse(setting.IsValid);
+        }
+
+        [TestMethod]
+        [TestCategory("Settings)")]
+        public void HotkeyModifier_ShiftAlt_IsValid()
+        {
+            var setting = new HotkeySetting { Name = "Foo", HasShiftModifier = true, HasAltModifier = true, Key1 = "I" };
+
+            Assert.IsTrue(setting.IsValid);
+        }
+
+        [TestMethod]
+        [TestCategory("Settings)")]
+        public void HotkeyModifier_ShiftCtrl_IsValid()
+        {
+            var setting = new HotkeySetting { Name = "Foo", HasShiftModifier = true, HasCtrlModifier = true, Key1 = "I" };
+
+            Assert.IsTrue(setting.IsValid);
+        }
+
+        [TestMethod]
+        [TestCategory("Settings)")]
+        public void HotkeyModifier_ShiftAltCtrl_IsValid()
+        {
+            var setting = new HotkeySetting { Name = "Foo", HasShiftModifier = true, HasAltModifier = true, HasCtrlModifier = true, Key1 = "I" };
+
+            Assert.IsTrue(setting.IsValid);
+        }
+
+        [TestMethod]
+        [TestCategory("Settings)")]
+        public void HotkeyModifier_Alt_IsValid()
+        {
+            var setting = new HotkeySetting { Name = "Foo", HasAltModifier = true, Key1 = "I" };
+
+            Assert.IsTrue(setting.IsValid);
+        }
+
+        [TestMethod]
+        [TestCategory("Settings)")]
+        public void HotkeyModifier_Ctrl_IsValid()
+        {
+            var setting = new HotkeySetting { Name = "Foo", HasCtrlModifier = true, Key1 = "I" };
+
+            Assert.IsTrue(setting.IsValid);
+        }
+
+        [TestMethod]
+        [TestCategory("Settings)")]
+        public void HotkeyModifier_AltCtrl_IsValid()
+        {
+            var setting = new HotkeySetting { Name = "Foo", HasAltModifier = true, HasCtrlModifier = true, Key1 = "I" };
+
+            Assert.IsTrue(setting.IsValid);
+        }
```

Comment:
This host of added tests seems like a good place to use parameterized tests

Reply:
Having a `private` test that is called that does the actual test? Something like `private bool TestHotkeyInputs( ... )`.